src/components/Widget/index.js

```diff
@@ -372,7 +372,8 @@ class Widget extends Component {
         start a new session.
         */
         const localId = this.getSessionId();
-        if (localId !== remoteId) {
+        const { userId } = customData || {};
```

Comment:
```suggestion
        const userId = customData?.userId;
```

Reply:
Since I'm only deconstructing one property instead of 2 then ok, we don't need the overkill by making it explicit.